FLEDGE.md

```diff
@@ -108,6 +109,8 @@ The browser will remain in an interest group for only a limited amount of time.
 
 #### 1.2 Interest Group Attributes
 
+The `priority` is used to select which interest groups are shown when the number of interest groups are limited by the `perBuyerGroupLimits` attribute of the auction config. If not specified, a `priority` of `0.0` is assigned.
```

Comment:
When there are multiple interest groups with the same `priority`, too many for the limit, perhaps say that the browser will select uniformly at random?

Reply:
Done